test/GPv2Settlement.test.ts

```diff
@@ -1,4 +1,5 @@
 import { ethers, waffle } from "@nomiclabs/buidler";
+import ERC20 from "@openzeppelin/contracts/build/contracts/ERC20PresetMinterPauser.json";
```

Comment:
nit: For unit tests we should probably use `waffle.deployMockContract`. This makes it easier to test weird behaviour with non-standard ERC20 tokens (such as returning `false` instead of reverting, etc).